.gitignore

```diff
@@ -1 +1,3 @@
 tmp
+bower_components
+out
```

Comment:
I would write this:

    /bower_components/
    /out/
    /tmp/

Changes:

  - leading slash to anchor to project root;
  - trailing slash for each directory; and
  - alphabetical order.